### PR TITLE
fixed issue - if test case is missing from run id then status of test…

### PR DESCRIPTION
**Pull request by nileshye**
fixed issue - if test case is missing from run id then status of test ids are not updating

### CODE REVIEW

File: testrail-connector/src/main/java/com/nullin/testrail/TestRailReporter.java [hunk 1]
```diff
@@ -230,105 +241,114 @@ public void reportResult(List<Map<String, Object>> properties) {
...
+					body.put("status_id", getStatus(resultStatus));
+					body.put("comment", new String(comment.toString().getBytes(StandardCharsets.ISO_8859_1),
+							StandardCharsets.UTF_8));
+					body.put("elapsed", elapsed);
+
+					props.add(body);
+					client.addResultForCase(runId, caseId, body);
+
+				} catch (Exception ex) {
+					logger.severe("For loop exception");
```
Comment (neofreko) [inline testrail-connector/src/main/java/com/nullin/testrail/TestRailReporter.java]: Please log the exception message to provide:
1. what action is failing
2. what's the cause

**Comment by neofreko** [comment]
Thanks for the PR @nileshye. I only see logging exception from Testrail which is good to provide information whether a case getting updated or not. Let me know if this PR contain more changes. Diff is hard to read :(

**Review by neofreko — APPROVED** [review]

**Review by nileshye — COMMENTED** [review]
added log line -

File: testrail-connector/src/main/java/com/nullin/testrail/TestRailReporter.java (hunk 1 above)
Comment (nileshye) [reply testrail-connector/src/main/java/com/nullin/testrail/TestRailReporter.java]: I have added logger line 
logger.severe("Test case "+case_id+ " is not pat of "+runId+" run id. Therefore status is not updated." + ex);

**Comment by nileshye** [comment]
by mistake i closed the PR. Apologies

**Pull request merged**